spec/javascripts/unit/AnalyticsSpec.js

```diff
@@ -70,30 +70,29 @@ describe("GOVUK.Analytics", function () {
     it("Should not call google analytics without a url", function () {
       $analyticsString = $("<div data-analytics='trackPageView'/>");
       $(document.body).append($analyticsString);
-      window.GOVUK.GDM.analytics.virtualPageViews.init();
+      window.GOVUK.GDM.analytics.virtualPageViews();
```

Comment:
I'm not sure I understand why these test cases are even in this application? Presumably they duplicate some in another app, that actually uses VPVs?!

Reply:
Yep, I think they were just copied over. For now they're useful to check that the toolkit is being pulled in correctly. We don't have any analytics tests in the toolkit yet, that was going to be the next tech debt step.

Reply:
I'm sure they're _that_ useful, as long as the toolkit's being tested _somewhere_ that'd be fine by me. I'd prefer to remove the duplicate code unless there's a strong concern that it needs to be here.

Reply:
More than happy to take them out (I've satisfied myself that the toolkit is working ok for this app).